_samples/library/couchdb.md

```diff
@@ -2,4 +2,5 @@
 redirect_to: https://hub.docker.com/_/couchdb/
 redirect_from:
 - /samples/couchdb/
+- /samples/couchdb_data_volumes/
```

Comment:
#16101 removes redirect only stub files. You can add this new `/samples/couchdb_data_volumes/` redirect after https://github.com/docker/docs/blob/59b19c2dce1f89e6a022ebde13762814d9e993cb/_redirects.yml#L202

cc @dvdksn

Reply:
Yeah so @craig-osterhout if you rebase this branch you can make the following change in `_redirects.yml`:

```diff
"https://hub.docker.com/_/couchbase/":
  - /samples/couchbase
  - /samples/library/couchbase
+ - /samples/couchdb_data_volumes/
```

and delete this file (`_samples/library/couchdb.md`)